nerdlandbot/scheduler/kerk_scheduler.py

```diff
@@ -0,0 +1,34 @@
+import os
+import discord
+import requests
+import time
+
+from datetime import datetime, timedelta
+from discord.ext import tasks
+
+from nerdlandbot.commands.GuildData import get_all_guilds_data, GuildData, get_guild_data
+from nerdlandbot.translations.Translations import get_text as translate
+from nerdlandbot.helpers.TranslationHelper import get_culture_from_context as culture
+from nerdlandbot.helpers.emoji import fist, church_emoji
+from nerdlandbot.helpers.log import warn
+
+@tasks.loop(minutes = 1.0)
+async def church_fights(bot):
+    guilds_data = await get_all_guilds_data()
+    day = datetime.now().day
+    hour = datetime.now().hour
```

Comment:
Change to only running datetime once, store the hour, check if it's equal to 16 **and if the minute equals 0**
This way we don't have to "waste cpu cycles" doing a bunch of stuff and then deciding we didn't actually need to do any of this